src/ImageSharp/Colors/Spaces/YCbCr.cs

```diff
@@ -117,13 +121,7 @@ public YCbCr(byte y, byte cb, byte cr)
         /// <inheritdoc/>
         public override int GetHashCode()
         {
-            unchecked
-            {
-                int hashCode = this.Y.GetHashCode();
-                hashCode = (hashCode * 397) ^ this.Cb.GetHashCode();
-                hashCode = (hashCode * 397) ^ this.Cr.GetHashCode();
-                return hashCode;
-            }
+            return GetHashCode(this);
```

Comment:
Resharper will actually warn if you don't do this since there is the potential for the hashcode to change during the lifetime of the object. I've not been crazy strict about it though, especially in the `IPackedPixel` implementations. Truth be told, I'm not sure it's something I should really be concerned about.

http://www.jetbrains.com/resharperplatform/help?Keyword=NonReadonlyMemberInGetHashCode